build.sbt

```diff
@@ -12,21 +11,26 @@ lazy val jvmSdkVersion = "1.1.0"
  */
 
 lazy val `commercetools-sunrise-payment` = (project in file("."))
-  .aggregate(`common`, `payone`)
+  .aggregate(`common`, `payone-adapter`)
   .settings(javaUnidocSettings ++ commonSettings : _*)
+  .settings(
+      libraryDependencies ++= Seq (
+        "com.novocode" % "junit-interface" % "0.11",
+        "org.assertj" % "assertj-core" % "3.4.1"
+      )
+  )
+  .dependsOn(`common`, `payone-adapter`)
 
 lazy val `common` = project
   .configs(IntegrationTest)
   .settings(commonSettings ++ commonTestSettings : _*)
   .settings(
     libraryDependencies ++= Seq(
-      "com.commercetools.sdk.jvm.core" % "commercetools-models" % jvmSdkVersion,
-      "com.commercetools.sdk.jvm.core" % "commercetools-java-client" % jvmSdkVersion,
-      "com.google.code.findbugs" % "jsr305" % "3.0.0"
```

Comment:
Btw now you are able to use it because of the JVM SDK having this as a dependency. I think it's fine, but I'm also fine on not depending on the JVM SDK for this dependency. Any outcome works for me :)